### DISCUSSION TITLE
Selected disabled option is not working on select

### DISCUSSION THREAD

**Discussion by ExsyDev** [livewire/livewire#1969] [4. General]
Hello, I have a code:

```
<select name="marital_status" id="marital_status" class="mt-1 block form-input rounded-md shadow-sm w-full" id="marital_status" wire:model.defer="state.marital_status" autocomplete="marital_status">
                <option selected disabled value="">Select one</option>
                <option value="married" @if($this->user->marital_status == 'married') selected @endif>Married</option>
                <option value="single" @if($this->user->marital_status == 'single') selected @endif>Single</option>
                <option value="divorced" @if($this->user->marital_status == 'divorced') selected @endif>Divorced</option>
            </select>
```

"Selected one" is not showing, if I remove: `wire:model.defer="state.marital_status" ` then disabled option show in select.

**Comment by iksaku** [comment]
While using Livewire, the `wire:model` attribute takes care of "selecting" the value according to your state, so you don't need to add `@if` statements for them. Also, you don't need to specify which `<option>` is `selected` by default, livewire will do that for you.

> Also, remember to not add duplicated `id` attributes to your `<select>` element.

Try this:
```html
<select wire:model.defer="state.marital_status" class="...">
    <option disabled>Select one</option>
    <option value="married">Married</option>
    <option value="single">Single</option>
    <option value="divorced">Divorced</option>
</select>
```

And in your component, try:
```php
class MyComponent extends Component {
    public $state = [
        // ...
        'marital_status' => null,
    ];
}
```

With that, your model will automatically take the first `<option>` (which has no value, so its `null`) as the default one.

**Reply by ExsyDev** [reply]
Not working :(

**Reply by iksaku** [reply]
You're right, just [Reproduced your example in a Playground](https://laravelplayground.com/#/snippets/8054e084-d197-45aa-9886-7857222f363c) and its indeed not working.

Would you mind opening a PR describing the issue and linking the playground above?

(Also, reference this discussion thread :wink:)

**Reply by MuathIbnHassan** [reply]
Hi, it worked with me when I did set the value in the disabled element to ```"null"```
yes, string "null" 🧐

```blade
<div>
  <select wire:model="state.marital_status">
    <option disabled value="null">Select one</option>
    <option value="married">Married</option>
    <option value="single">Single</option>
    <option value="divorced">Divorced</option>
  </select>
</div>
```

```php
public $state = [
    'marital_status' => null
];
```

see https://laravelplayground.com/#/snippets/24b329d5-26f2-4776-9f24-d8bc2cd5e7af
